content/en/docs/reference/glossary/service.md

```diff
@@ -11,8 +11,10 @@ tags:
 - fundamental
 - core-object
 ---
-An abstract way to expose an application running on a set of {{< glossary_tooltip text="Pods" term_id="pod" >}} as a network service.
+A method for exposing an network application that is running as one or more {{< glossary_tooltip text="Pods" term_id="pod" >}} in your cluster.
 
 <!--more-->
 
- The set of Pods targeted by a Service is (usually) determined by a {{< glossary_tooltip text="selector" term_id="selector" >}}. If more Pods are added or removed, the set of Pods matching the selector will change. The Service makes sure that network traffic can be directed to the current set of Pods for the workload.
+The set of Pods targeted by a Service is (usually) determined by a {{< glossary_tooltip text="selector" term_id="selector" >}}. If more Pods are added or removed, the set of Pods matching the selector will change. The Service makes sure that network traffic can be directed to the current set of Pods for the workload.
+
+Kubernetes Services use IP networking (IPv4, IPv6, or both).
```

Comment:
Em... There is an "external" type of Service.

Reply:
Good point. The original had that drawback but we shouldn't prolong it.